Refactor and docker support

dhvcc opened the pull request:
Fixed name shadowing and made table creation run automatically on startup
Added Dockerfile for an app and docker-compose with postgres 12
Created `.env` example

dhvcc committed:
Added Docker support

Added basic Dockerfile and docker-compose with postgres 12
Added .env.example

dhvcc committed:
Refactor

Fixed name shadowing in app.py
Made setup_db run automatically on startup

dhvcc committed:
Merge branch 'feature-docker' into main

dhvcc commented:
Сейчас в ридми описание ещё добавлю

nomilkinmyhome commented on Dockerfile:
```diff
@@ -0,0 +1,18 @@
+FROM python:3.8-slim as base
```
Я тестила на версии 3.7. Возможно на 3.8 тоже работает, но насколько мне известно ботл нестабилен на этой версии питона, так что лучше замени на 3.7

nomilkinmyhome commented on src/__main__.py:
```diff
@@ -1,5 +1,5 @@
 from src.app import bot
 
-if __name__ == '__main__':
+if __name__ == "__main__":
```
По всему проекту одинарные кавычки ведь.

dhvcc replied:
А, да, забыл убрать

dhvcc committed:
Small updates

Changed python version to 3.7 in Dockerfile
Updated README
Formatted imports in app.py
Single-quoted strings

nomilkinmyhome commented on src/app.py:
```diff
@@ -6,6 +6,8 @@
 from src.blueprints import bps
 from src.config import BOT_TOKEN
 from src.middlewares.no_bot_middleware import NoBotMiddleware
+from vkbottle.tools.dev_tools.loop_wrapper import LoopWrapper
```
Отсортируй импорты, пожалуйста. Можешь прогнать flake8

dhvcc replied:
Блин, не увидел

dhvcc commented:
Делал форс пуши, что бы историю не ляпать
